execution_engine/src/core/resolvers/v1_resolver.rs

```diff
@@ -229,6 +229,10 @@ impl ModuleImportResolver for RuntimeModuleImportResolver {
                 Signature::new(&[ValueType::I32; 1][..], Some(ValueType::I32)),
                 FunctionIndex::NewDictionaryFuncIndex.into(),
             ),
+            "casper_dictionary_read" => FuncInstance::alloc_host(
+                Signature::new(&[ValueType::I32; 3][..], Some(ValueType::I32)),
+                FunctionIndex::DictionaryReadFuncIndex.into(),
+            ),
```

Comment:
move this up next to casper_dictionary_get

Reply:
Addressed in https://github.com/casper-network/casper-node/pull/3161/commits/862d2b060f8391957da23d76025c2b8a6fd92c5f